### PR TITLE
rbenv-bundler 1.0.1

### PR DESCRIPTION
**Pull request by carsomyr**
Created with `brew bump-formula-pr`.

### CODE REVIEW

File: Formula/rbenv-bundler.rb [hunk 1]
```diff
@@ -1,15 +1,12 @@
...
+  url "https://github.com/carsomyr/rbenv-bundler/archive/1.0.1.tar.gz"
+  sha256 "6840d4165242da4606cd246ee77d484a91ee926331c5a6f840847ce189f54d74"
   license "Apache-2.0"
-  revision 1
   head "https://github.com/carsomyr/rbenv-bundler.git"
 
-  bottle do
-    sha256 cellar: :any_skip_relocation, all: "d99c88c9492de9f01e61f3ba4d8930dbf1b607ac50c60d23c1e2e8963bd43419"
-  end
+  bottle :unneeded
```
Comment (nandahkrishna) [inline Formula/rbenv-bundler.rb:9]: Why? Please don't edit the bottle block.

Comment (carsomyr) [reply Formula/rbenv-bundler.rb]: Can you point out what's making the CI fail? Seems like it's bottle related.

**Comment by nandahkrishna** [comment]
Thanks for reverting the change. The `ubuntu-latest` failure can be ignored, that will be fixed by an update to our test bot.

**Comment by carsomyr** [comment]
@nandahkrishna Can you merge this?

**Comment by nandahkrishna** [comment]
CI isn't done yet, bottles are yet to be built on 11 (Intel), 10.15 and 10.14 – you can see that the job has been queued. The PR will be merged after CI is done.

**Review by nandahkrishna — APPROVED** [review]

**Pull request closed without merging**